akka-http/src/main/scala/akka/http/scaladsl/common/NameReceptacle.scala

```diff
@@ -16,19 +17,35 @@ class NameReceptacle[T](val name: String) {
   def as[B] = new NameReceptacle[B](name)
   def as[B](unmarshaller: Unmarshaller[T, B])(implicit fsu: FSU[T]) =
     new NameUnmarshallerReceptacle(name, fsu.transform[B](implicit ec => implicit mat => { _ flatMap unmarshaller.apply }))
+  /** Symbolic alias for [[optional]]. */
```

Comment:
Too bad we don't have https://dotty.epfl.ch/api/scala/annotation/alpha.html yet

Reply:
There it was, I knew something had been done to improve this.